### PR TITLE
15.0 mig website rma

### PR DESCRIPTION
**Pull request by ghost**
minor changes for migration

### CODE REVIEW

**Comment by victoralmau** [comment]
Can you check the commit history? It seems that there is something wrong and that a translation has been mixed with the migration commit.

Remember that the last 2 commits must be:
- [IMP] website_rma: black, isort, prettier
- [MIG] website_rma: Migration to 15.0

**Comment by ghost** [comment]
jetzt mit richtiger History.

File: website_rma/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "views/website_rma_portal_templates.xml",
+        "views/website_templates.xml",
+        "data/website_data.xml",
+    ],
+    "assets": {
+        "web.assets_frontend": [
+            "/website_rma/static/src/js/website_rma.tour.js",
+            "/website_rma/static/src/js/website_rma.js",
+        ],
+    },
```
Comment (victoralmau) [inline website_rma/__manifest__.py]:
```suggestion
"assets": {
        "web.assets_frontend": [
            "/website_rma/static/src/js/website_rma.js"
        ],
        "web.assets_tests": [
            "/website_rma/static/src/js/website_rma.tour.js"
        ]
    },
```

File: website_rma/views/request_rma_form.xml [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                <li>
+                    <i class="fa fa-phone fa-fw mr-2" />
+                    <span class="o_force_ltr">+1 (650) 555-0111</span>
+                </li>
+                <li>
+                    <i class="fa fa-1x fa-fw fa-envelope mr-2" />
+                    <span>info@yourcompany.example.com</span>
+                </li>
+            </ul>
+        </div>
```
Comment (victoralmau) [inline website_rma/views/request_rma_form.xml]:
```suggestion
<!--Similar to existing website.company_description of v14!-->
<address itemscope="itemscope" itemtype="http://schema.org/Organization">
        <!-- TODO widget contact must add itemprop attributes -->
        <div t-field="res_company.partner_id" t-options='{
                "widget": "contact",
                "fields": ["name", "address", "phone", "mobile", "email"]}'/>
        <t t-if="not res_company.google_map_img()">
            <span class="fa fa-map-marker fa-fw mt16" role="img" aria-label="Address" title="Address"/> <a t-att-href="res_company.google_map_link()" target="_BLANK"> Google Maps</a>
        </t>
    </address>
```

File: website_rma/views/request_rma_form.xml [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+                                        </div>
+                                    </div>
+                                </form>
+                            </div>
+                            <div
+                                class="oe_structure mt-2"
+                                id="oe_structure_website_rma_form_request_2"
+                            />
+                        </div>
+                        <t t-call="website_rma.company_description" />
```
Comment (victoralmau) [inline website_rma/views/request_rma_form.xml]:
```suggestion
                        <div class="col-lg-4">
                            <t t-call="website_rma.company_description" />
                        </div>
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Functional review OK.

Ping @chienandalu

File: website_rma/views/request_rma_form.xml (hunk 3 above)
Comment (ghost) [reply website_rma/views/request_rma_form.xml]: @victoralmau
fixed

**Review by victoralmau — APPROVED** [review]
Code and functional review OK.

I am not entirely sure if `company_description` block is the right way or it is better to approach it in another way. What do you think @chienandalu ?

File: website_rma/views/request_rma_form.xml (hunk 3 above)
Comment (chienandalu) [inline website_rma/views/request_rma_form.xml]: Well, it's useless to create this additional view with demo data. Odoo's approach now is that the user simply edits the page in the website and thus a custom website is created. So for me, this block can simply go away and let the user custom the page as he please

**Comment by ghost** [comment]
@victoralmau:
"company_description" is removed

File: website_rma/views/request_rma_form.xml [hunk 4]
```diff
@@ -124,9 +124,11 @@
                                                 class="form-control o_website_form_input"
                                                 name="description"
                                                 required=""
-                                            ><t
+                                            >
+                                                <t
                                                     t-esc="request.params.get('description', '')"
-                                                /></textarea>
+                                                />
+                                            </textarea>
```
Comment (chienandalu) [inline website_rma/views/request_rma_form.xml]: Beware of reformatting `<textarea>` tags, as they behave like `<pre>` so you've got this empty space from the xml declaration:

image

**Review by chienandalu — CHANGES_REQUESTED** [review]
Other than this, tested :+1:

**Review by chienandalu — APPROVED** [review]
Thanks for your work @nikolausweingartmair !

**Pull request merged**